translations/pt-BR/README.md

```diff
@@ -109,155 +107,153 @@ Using a virtual terminal is not really convenient. For example, if you want to e
 
 ## Modo não-interativo
 
-In non-interactive mode, the shell reads commands from a file or a pipe and executes them. When the interpreter reaches the end of the file, the shell process terminates the session and returns to the parent process.
+No modo não-interativo, o shell recebe comandos de um arquivo ou um _pipe_ e executa eles. Quando o interpretador chega no final do arquivo, a sessão de processamento do shell é terminada e o processo anterior é retornado.
 
-Use the following commands for running the shell in non-interactive mode:
+Use os seguintes comandos para executar o shell em modo não-interativo:
 
     sh /path/to/script.sh
```

Comment:
We are explaining `bash` right? So let's avoid the use of `sh`
We can use the `. /path/to/script.sh` will do the same as `bash` command